render/wlr_renderer.c

```diff
@@ -23,6 +23,15 @@ void wlr_renderer_end(struct wlr_renderer *r) {
 	r->impl->end(r);
 }
 
+void wlr_renderer_clear(struct wlr_renderer *r, float red, float green,
+		float blue, float alpha) {
+	r->impl->clear(r, red, green, blue, alpha);
+}
+
+void wlr_renderer_scissor(struct wlr_renderer *r, struct wlr_box *box) {
+	r->impl->scissor(r, box);
```

Comment:
Check for null

Reply:
Thta's already done in the implementation. Or do you mean check for NULL `impl->scissor`? I don't know if it makes sense, all of the `impl` fields are mandatory for now. Maybe it would make sense to `assert` all of these when creating the renderer.